src/components/Header/Header/Header.tsx

```diff
@@ -0,0 +1,48 @@
+import { Link } from "react-router-dom";
+import Logo from "@/assets/svg/Logo";
+import styles from "./Header.module.scss";
+import useGet from "../../../hooks/useGet";
+import { useEffect, useState } from "react";
+import { END_POINT } from "../../../constants";
+import { UserData } from "@/constants/index.types";
+
+interface HeaderProps {
+  headerPosition: React.CSSProperties;
```

Comment:
외부에서 어떤 스타일을 주입할지는 컴포넌트 입장에서는 모르기때문에 특정 스타일 이름을 나타내기 보다는 일반적인 이름으로 표현하면 좋을 것 같구요!
```suggestion
  headerStyle: React.CSSProperties;
```

Reply:
아예 `position: 'fixed' | 'normal'` 이런식으로 props를 받고 내부에서 어떤 값이 들어오냐에 따라서 className으로 스타일링을 해줘도 좋을 것 같아요!

Reply:
이런경우 enum 사용이 가능할까요?

Reply:
네! enum을 사용해도 되고, 인라인으로 유니온 리터럴 타입을 만들어줘도 되고, 상수를 활용해도 됩니다!